bitnami/airflow/Chart.yaml

```diff
@@ -32,4 +32,4 @@ name: airflow
 sources:
   - https://github.com/bitnami/bitnami-docker-airflow
   - https://airflow.apache.org/
-version: 10.3.3
+version: 10.3.4
```

Comment:
A new version was released (`11.0.0`) you need to rebase/fix the conflict and then bump to `11.0.1`

Reply:
ok!